carbon-design-system/ibm-security
# build(storybook): upgrade packages and configuration

## Proposed changes

- Upgrade to latest Storybook packages
- Migrate configuration as per https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#to-mainjs-configuration
- Use native Storybook controls for theme addon
- Remove `addon-centered` and update layout configuration as it's now handled natively
- Remove unused packages no longer required with migration
